examples/generate-trades-history-csv.py

```diff
@@ -37,20 +37,20 @@ def data(start, end, ofs):
 k = krakenex.API()
 k.load_key('kraken.key')
 
-data = []
+data2 = []
 count = 0
-for i in range(1,11):
-    start_date = datetime.datetime(2016, i+1, 1)
-    end_date = datetime.datetime(2016, i+2, 29)
+for i in range(1,2):
```

Comment:
The loop was specifically for retrieving the data on a per-month basis. I don't remember why this was done - maybe it was required by the tax form filling script (also in `exmples`)?..